src/extension/geofence/src/main/java/org/geoserver/geofence/GeofenceAccessManager.java

```diff
@@ -178,22 +179,26 @@ public WorkspaceAccessLimits getAccessLimits(Authentication user, WorkspaceInfo
         }
 
         // further logic disabled because of https://github.com/geosolutions-it/geofence/issues/6
-        return new WorkspaceAccessLimits(DEFAULT_CATALOG_MODE, true, false);
+        // (gone)
+        final boolean readable = true;
+        final boolean writable = false;
+        return new WorkspaceAccessLimits(DEFAULT_CATALOG_MODE, readable, writable);
     }
 
     /** We expect the user not to be null and not to be admin */
     private boolean isWorkspaceAdmin(Authentication user, String workspaceName) {
         LOGGER.log(Level.FINE, "Getting admin auth for Workspace {0}", workspaceName);
+        final String username = user.getName().toLowerCase();
```

Comment:
Why we are going to convert the username to lowercase?

Reply:
with georchestra, the username keeps the case the user logged in with (ie, a user registered as "user" can also log in as "User" or "uSer" for example and that will be the case we get with user.getName()).

Yes this is not relevant here.

Reply:
lowercase is no longer used.